apk-generator/v2/app/generator/generator.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, config, via_api=False, identifier=None, task_handle=None):
             self.identifier = identifier
         self.task_handle = task_handle
         self.update_status('Starting the generator')
+        self.update_status('Initializing the variables')
```

Comment:
This is not a valid status step. Initialising variables is part of every program. Not an actual _process_.